### PR TITLE
Support `SELECT` with empty target list

### LINKED ISSUE #13: "SELECT FROM table" not supported
`SELECT FROM table` is valid syntax - the select list may be empty.

### PR DESCRIPTION
**Pull request by rvanvelzen**
Fix #13

### CODE REVIEW

**Comment by rvanvelzen** [comment]
I realize this may not be an optimal solution, but it _does_ work.

File: src/sad_spirit/pg_builder/Parser.php [hunk 1]
```diff
@@ -22,6 +22,8 @@
 
 use Psr\Cache\CacheItemPoolInterface;
 use Psr\Cache\InvalidArgumentException;
+use sad_spirit\pg_builder\exceptions\SyntaxException;
+use sad_spirit\pg_builder\nodes\lists\TargetList;
```
Comment (sad-spirit) [inline src/sad_spirit/pg_builder/Parser.php:26]: Please don't `use` the package classes, use relative namespaces as everywhere else

**Comment by rvanvelzen** [comment]
I've decided to take a slightly more straightforward approach.

**Comment by sad-spirit** [comment]
Yes, that's a step in a right direction! Unfortunately you don't do all the checks I described above, so some constructs will not work, try e.g.
```sql
select union (select) intersect select limit 1
```

**Comment by rvanvelzen** [comment]
> Unfortunately you don't do all the checks I described above

Sorry, I don't see the checks that you mention here :confused:

**Comment by sad-spirit** [comment]
I added a review to your previous commit, github shows it inline for me in this PR, but probably not for you. Go figure...

Copying the most relevant part:
> Looking at the `SelectStmt` and a few next grammar productions in `gram.y`, I think it will be sufficient to test for
> - end of input
> - closing parenthesis `)`
> - a reserved keyword `into`, `from`, `where`, `group`, `having`, `window`, `union`, `intersect`, `except`, `order`, `limit`, `offset`, `fetch`, `for`
>
> Everything else after `SELECT` is probably a part of the target list.

**Pull request merged**